review/src/ReviewConfig.elm

```diff
@@ -44,6 +45,14 @@ config =
         |> Rule.ignoreErrorsForDirectories [ "tests/" ]
     , NoRedundantConcat.rule
     , NoRedundantCons.rule
+    , NoUnsortedRecordFields.rule
+        |> Rule.ignoreErrorsForDirectories [ "tests/" ]
+        |> Rule.ignoreErrorsForDirectories [ "src/Page" ]
+        |> Rule.ignoreErrorsForFiles [ "src/Data/Impact/Definition.elm" ]
+        |> Rule.ignoreErrorsForFiles [ "src/Views/Page.elm" ]
```

Comment:
Note to reviewers: these are excluded because they feature record types used as constructors where the order of fields is important and so we can't reorder them.